main_test.go

```diff
@@ -0,0 +1,182 @@
+		},
+		{
+			testName:       "invalid uid",
+			filters:        []string{"uid>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>0"},
+			expectedFilter: tracee.Filter{},
+			expectedError:  errors.New("invalid UID value: >"),
+		},
+		{
+			testName:       "invalid uid",
+			filters:        []string{"uid=a"},
+			expectedFilter: tracee.Filter{},
+			expectedError:  errors.New("invalid UID value: a"),
+		},
+		{
+			testName:       "invalid uid",
+			filters:        []string{"uid=4294967296"},
+			expectedFilter: tracee.Filter{},
+			expectedError:  errors.New("invalid UID value: 4294967296"),
+		},
+		{
+			testName:       "invalid uid",
+			filters:        []string{"uid=-1"},
+			expectedFilter: tracee.Filter{},
+			expectedError:  errors.New("invalid UID value: -1"),
+		},
+		{
+			testName:       "invalid uid",
+			filters:        []string{"uid=1 1"},
+			expectedFilter: tracee.Filter{},
+			expectedError:  errors.New("invalid UID value: 1 1"),
+		},
+		{
+			testName:       "invalid uid",
+			filters:        []string{"uid="},
+			expectedFilter: tracee.Filter{},
+			expectedError:  errors.New("invalid UID value: "),
+		},
+		{
+			testName: "invalid uid",
+			filters: []string{"uid=	"},
+			expectedFilter: tracee.Filter{},
+			expectedError: errors.New("invalid UID value: 	"),
+		},
+		{
+			testName:       "invalid uid",
+			filters:        []string{"uid=AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"},
+			expectedFilter: tracee.Filter{},
+			expectedError:  errors.New("invalid UID value: AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"),
+		},
+		{
+			testName:       "invalid uid",
+			filters:        []string{"uid=%s"},
+			expectedFilter: tracee.Filter{},
+			expectedError:  errors.New("invalid UID value: %s"),
+		},
+		{
+			testName:       "invalid uid",
+			filters:        []string{"uid=\t\t\t"},
+			expectedFilter: tracee.Filter{},
+			expectedError:  errors.New("invalid UID value: \t\t\t"),
+		},
+		{
+			testName:       "invalid uid",
+			filters:        []string{"uid=0\t"},
+			expectedFilter: tracee.Filter{},
+			expectedError:  errors.New("invalid UID value: 0\t"),
+		},
+		{
+			testName: "uid=0",
+			filters:  []string{"uid=0"},
+			expectedFilter: tracee.Filter{
+				UIDFilter: tracee.UIDFilter{
+					Equal:    []uint32{0},
+					NotEqual: []uint32{},
+					Greater:  4294967295,
+					Less:     0,
+				},
+			},
+			expectedError: nil,
+		},
+		{
+			testName: "uid!=0",
+			filters:  []string{"uid!=0"},
+			expectedFilter: tracee.Filter{
+				UIDFilter: tracee.UIDFilter{
+					Equal:    []uint32{},
+					NotEqual: []uint32{0},
+					Greater:  0,
+					Less:     4294967295,
+				},
+			},
+			expectedError: nil,
+		},
+		{
+			testName: "uid>0",
+			filters:  []string{"uid>0"},
+			expectedFilter: tracee.Filter{
+				UIDFilter: tracee.UIDFilter{
+					Equal:    []uint32{},
+					NotEqual: []uint32{},
+					Greater:  0,
+					Less:     0,
```

Comment:
shouldn't 'Less' have the 4294967295 default value as it is not set by this filter?